server_test.go

```diff
@@ -907,7 +907,7 @@ type Server struct {
 
 // NewServer returns a new test server instance.
 func NewServer() *Server {
-	return &Server{influxdb.NewServer()}
+	return &Server{influxdb.NewServer("X.X")}
```

Comment:
I'll refactor to allow only the handler being passed the version.  makes more sense after getting feedback.